events/tests/UNITTESTS/CMakeLists.txt

```diff
@@ -0,0 +1,4 @@
+# Copyright (c) 2021 ARM Limited. All rights reserved.
```

Comment:
one thing, do we keep UNITTESTS with capital letters? It was because of the old tools, we do not need it anymore with CMake, do we ?

Reply:
Asking now :-) It's cosmetic change but I assume we don't need to have t hem with capital letters, or is there a reason?

Reply:
It is because of old tools, once we deprecate old tools we can able to rename to unittests with small letters